.github/workflows/style.yaml

```diff
@@ -117,22 +141,3 @@ jobs:
           version: v1.62.0
           args: --timeout=5m
 
-  lint:
```

Comment:
This was doing nothing and we're already covering what it would handle via the other Jobs.